stackrox/stackrox
# Rewrite metadata service in TypeScript

## Description

Increase type safety with compile-time checks, especially as prerequisite for possible future call by component instead of saga.

## Checklist

## Testing Performed

1. `yarn build` in ui
2. `yarn start` in ui

    * See version on Help menu
        ![version-Help](https://user-images.githubusercontent.com/11862657/168821369-16aa2bd3-e527-43a8-b633-e23382783e7a.png)

    * See version on Clusters page
        ![version-Clusters](https://user-images.githubusercontent.com/11862657/168821433-8810b663-6e7e-440d-a8df-050e8cf1bded.png)
